web-common/src/test/java/edu/unc/lib/boxc/web/common/services/AccessCopiesServiceTest.java

```diff
@@ -0,0 +1,180 @@
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+/**
+ * @author lfarrell
+ */
+public class AccessCopiesServiceTest  {
+    private PermissionsHelper helper;
+
+    private ContentObjectSolrRecord mdObject;
+
+    private ContentObjectSolrRecord mdObjectImg;
+
+    private AccessGroupSet principals;
+
+    private AccessCopiesService accessCopiesService;
+
+    @Mock
+    private AccessControlService accessControlService;
+    @Mock
+    private AccessRestrictionUtil restrictionUtil;
+    @Mock
+    private SolrSettings solrSettings;
+    @Mock
+    private FacetFieldUtil facetFieldUtil;
+    @Mock
+    private SolrClient solrClient;
+    @Mock
+    private QueryResponse queryResponse;
+    @Mock
+    private SolrDocumentList solrDocumentList;
+
+    @Before
+    public void init() throws IOException, SolrServerException {
+        initMocks(this);
+
+        mdObject = new ContentObjectSolrRecord();
+        mdObject.setId("9ef8d1c5-14a1-4ed3-b0c0-6da67fa5f6d1");
+        List<String> datastreams = Collections.singletonList(
+                ORIGINAL_FILE.getId() + "|application/pdf|file.pdf|pdf|766|urn:sha1:checksum|");
+        mdObject.setDatastream(datastreams);
+
+        mdObjectImg = new ContentObjectSolrRecord();
+        mdObjectImg.setId("27f8d1c5-14a1-4ed3-b0c0-6da67fa5f6d1");
+        List<String> imgDatastreams = Collections.singletonList(
+                ORIGINAL_FILE.getId() + "|image/png|file.png|png|766|urn:sha1:checksum|");
+        mdObjectImg.setDatastream(imgDatastreams);
+
+        principals = new AccessGroupSetImpl("group");
+
+        helper = new PermissionsHelper();
+        helper.setAccessControlService(accessControlService);
+
+        Properties searchProps = new Properties();
+        searchProps.load(this.getClass().getResourceAsStream("/search.properties"));
+        SearchSettings searchSettings = new SearchSettings();
+        searchSettings.setProperties(searchProps);
+
+        accessCopiesService = new AccessCopiesService();
+        accessCopiesService.setPermissionsHelper(helper);
+        accessCopiesService.setAccessRestrictionUtil(restrictionUtil);
+        accessCopiesService.setSolrSettings(solrSettings);
+        accessCopiesService.setSearchSettings(searchSettings);
+        accessCopiesService.setFacetFieldUtil(facetFieldUtil);
+        accessCopiesService.setSolrClient(solrClient);
+
+        when(solrClient.query(any(SolrQuery.class))).thenReturn(queryResponse);
+        when(queryResponse.getResults()).thenReturn(solrDocumentList);
+        when(solrDocumentList.getNumFound()).thenReturn(1L);
+    }
+
+    @Test
+    public void testHasViewAblePdf() {
```

Comment:
"Viewable" is one word, so "Able" shouldn't be capitalized in any of these tests